src/main/scala/rocket/PMP.scala

```diff
@@ -161,6 +161,25 @@ class PMPChecker(lgMaxSize: Int)(implicit p: Parameters) extends CoreModule()(p)
     val hit = pmp.hit(io.addr, io.size, lgMaxSize, prevPMP)
     val ignore = default && !pmp.cfg.l
     val aligned = pmp.aligned(io.addr, io.size, lgMaxSize, prevPMP)
+
+    cover(!default && pmp.cfg.a === 0x0, "The cfg access is set to no access",    "Cover PMP access mode setting")
+    cover(!default && pmp.cfg.a === 0x1, "The cfg access is set to TOR access",   "Cover PMP access mode setting")
+    cover(!default && pmp.cfg.a === 0x2, "The cfg access is set to NA4 access",   "Cover PMP access mode setting")
+    cover(!default && pmp.cfg.a === 0x3, "The cfg access is set to NAPOT access", "Cover PMP access mode setting")
```

Comment:
Maybe express these four with a loop? Something like

    for ((name, idx) <- Seq("no", "TOR", "NA4", "NAPOT").zipWithIndex)
      cover(!default && pmp.cfg.a === idx, s"The cfg access is set to $name access", "Cover PMP access mode setting")